runtime/src/main/cpp/Memory.cpp

```diff
@@ -1650,6 +1811,7 @@ void Kotlin_native_internal_GC_stop(KRef) {
 #if USE_GC
   if (memoryState->toFree != nullptr) {
     GarbageCollect();
+    // TODO: what shall we do with toRelease here? Just leak?
```

Comment:
Btw, this GC.start/stop feature doesn't seem to work properly.
Consider e.g.
```
kotlin.native.internal.GC.stop()
Any().freeze()
```

Reply:
Yeah, more testing is needed, not sure if in this PR.